### DISCUSSION TITLE
Add Chains: Binance Smart Chain (BSC) and testnet

### DISCUSSION THREAD

**Discussion by taycaldwell** [wevm/wagmi#914] [Ideas]
Feature request to add chain(s):

- Binance Smart Chain (BSC)
- Binance Smart Chain Testnet

I can open a PR to add it once I get the +1.

**Comment by sagargulati** [comment]
+1

**Comment by 0xPr0f** [comment]
+1

**Comment by aiekseu** [comment]
+1

**Comment by sunopar** [comment]
Is there any update?

**Comment by ghost** [comment]
+1
having trouble figuring out how to connect to the BSC and Cronos chains...
halp

**Comment by taycaldwell** [comment]
Hello fam,

I made a PR to add support here: https://github.com/wagmi-dev/wagmi/pull/1197

**Comment by Mubasher16** [comment]
+1

**Comment by GrizzlyWebDev** [comment]
Hello all! I had this same question a while ago and figured it out, here is a link to my solution. 

https://github.com/wagmi-dev/wagmi/discussions/1051

**Comment by dolgoonnn** [comment]
+1

**Reply by ghost** [reply]
hey buddy check out Grizzly's answer above

**Comment by armatrix** [comment]
+1

**Comment by jxom** [comment]
With the introduction of `0.9.x`, wagmi now comes with the `bsc` & `bscTestnet` chains built-in (re-exported from our [`@wagmi/chains` reference repository](https://github.com/wagmi-dev/references/tree/main/packages/chains)). Read more [here](https://wagmi.sh/react/chains).

**Reply by ghost** [reply]
Cool! It was fun to learn how to add a custom chain, but hey, working out of the box is great! thanks fellas

**Reply by jxom** [reply]
Sweet!

**Reply by wtakaman** [reply]
+1

**Comment by Droppix** [comment]
+1
